keyboards/bbrfkr/dynamis/config.h

```diff
@@ -26,9 +26,9 @@
 #define PRODUCT dynamis
 
 /* key matrix */
-#define MATRIX_ROWS 10
+#define MATRIX_ROWS 11
 #define MATRIX_COLS 7
-#define MATRIX_ROW_PINS { B6, B4, D6, D5, D1, C6, B5, D7, D4, D0 }
+#define MATRIX_ROW_PINS { B6, B4, D6, D5, D1, C6, B5, D7, D4, D0, D2 }
```

Comment:
Is D2 actually being used here? Or just being used as a place holder? 
```suggestion
#define MATRIX_ROW_PINS { B6, B4, D6, D5, D1, C6, B5, D7, D4, D0 }
```

Reply:
D2 is for the encoder. I think that the default via keymap for encoder cannot be defined if this doesn't exist.

Reply:
You can undefine and redefine the values for the via keymap, actually.  

Which is the ideal option here, since you're creating blank spots in the layout that are ONLY used for the encoders.  


Also, you should use `NO_PIN` instead of assigning a pin here.

Reply:
I see. I replaced `D2` to `NO_PIN`.